src/main/gui/main-window.ts

```diff
@@ -379,6 +368,28 @@ export const createMainWindow = async (args: OpenArguments) => {
             return;
         }
 
+        if (isMac) {
+            if (process.env.STARTUPFILE) {
```

Comment:
Could we use a global variable instead of an untyped process env variable?

Reply:
done